src/style.css

```diff
@@ -1,3 +1,78 @@
 body {
+  font-family: Roboto, system-ui;
+  font-weight: 500;
+  font-size: 12px;
+  color: #000;
   margin: 0;
+
+  --header-height: 60px;
+  --blue-color: #00acdc;
+}
+
+.header {
+  padding: 0 50px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.header__logo {
+  height: 40px;
+  width: 40px;
+}
+
+.header__img {
+  height: 100%;
+  width: 100%;
+  display: inline-block;
+}
+
+.menu {
+  align-items: center;
+  display: flex;
+}
+
+.menu__list {
+  padding: 0;
+  margin: 0;
+  display: flex;
+}
+
+.menu__item {
+  margin-right: 20px;
```

Comment:
Be consistent with your vertical margins. Here you're using a right margin on '.menu__item'. If you need spacing between items, consider using gap property on the parent '.menu__list' with flex display.